source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,481 @@
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16150.0.0  16150.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16500.0.0  16500.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16810.0.0  16810.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18080.0.0  18080.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18620.0.0  18620.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18860.0.0  18860.0.0
+
+The information shown in the `Urn` column lists the
+`Publisher:Offer:Sku:Version` for each image available and this is the
+information to use to create the |CL| VM. Since we are creating a |CL| basic
+VM, highlight the `clear-linux-project:clear-linux-os:basic:` string and copy
+it to your clipboard. For the version you can use the label `latest` instead
+of referencing a specific version, which is what we will do when we create our
+VM in a moment.
+
+Create a MS Azure resource group
+********************************
+
+With all the information gathered, we need to create a resource group to
+manage multiple resources within MS Azure for our |CL| VM. To learn more about
+resource groups, visit the `Azure Resource Manager overview`_ for an overview
+and detailed description of resources within MS Azure.
+
+To create our new resource group, run the :command:`az` command shown below to
+create a resource group named `ClearResourceGroup` and locate it in the
+`westus` region.
+
+.. code-block:: console
+
+   az group create -n ClearResourceGroup -l westus
+
+When the command has completed, the output from this command is similar to
+the following:
+
+.. code-block:: console
+
+   {
+     "id": "/subscriptions/{unique-id}/resourceGroups/ClearResourceGroup",
+     "location": "westus",
+     "managedBy": null,
+     "name": "ClearResourceGroup",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
+Create a |CL| virtual machine
+*****************************
+
+To create a new |CL| VM, run the following :command:`az` command using the
+URN `:clear-linux-project:clear-linux-os:basic:latest` that we located earlier
+in our search for the |CL| images available in the MS Azure marketplace:
+
+.. code-block:: console
+
+   az vm create --resource-group ClearResourceGroup --name ClearVM --image clear-linux-project:clear-linux-os:basic:latest --generate-ssh-keys
+
+.. note::
+
+   If you have already defined your public/private SSH key pair and they are
+   stored in your :file:`$HOME/.ssh` directory, you do not need to include the
+   :parameter:`--generate-ssh-keys` parameter.
+
```

Comment:
> +   If you have already defined your public/private SSH key pair and they are
> +   stored in your :file:`$HOME/.ssh` directory, you do not need to include the
> +   :parameter:`--generate-ssh-keys` parameter.

Can we offer a quick how-to on creating public/private SSH key pair ?

Reply:
I agree with Tom. We need to add that quick guide to the network maintenance guide.  Once that is added we can modify this section to cross-reference it.